index.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.contact-form {
+  max-width: 800px;
```

Comment:
I think if you use `rem` instead of `px` then the page adapts better when the base font size is changed in the user's browser.